tests/console/check_system_info.pm

```diff
@@ -61,6 +61,13 @@ sub run {
         my $out = script_output("zypper lr | grep -i $name", 200, proceed_on_failure => 1);
         die "zypper lr command output does not include $name" if ($out eq '');
     }
+
+    # Checked to get expected buildID with proxy scc upgrade
+    if ((get_var('SCC_URL', "") =~ /proxy/) && !get_var("MEDIA_UPGRADE") && get_var("BUILD_SLE") && !get_var("ONLINE_MIGRATION")) {
+        my $build_id = get_var("BUILD_SLE");
+        my $build    = script_output("zypper lr --url | grep -i $build_id", 200, proceed_on_failure => 1);
+        die "System does not upgrade to expected build ID: $build_id" if ($build eq '');
```

Comment:
I'm not sure if this check really works because this only grep build id in the output of zypper lr. Maybe we need find a more reliable method to do this check.

Reply:
Yutao,

I suggest to check /etc/SUSEConnect and run zypper patch to make sure you have already updated the system

Reply:
Those operation has been done by other test modules

Reply:
But if you check your system is update to date, zypper lr is not enough, am I right? it can only show active repos there.

Reply:
For build ID, we only can get from zypper lr --url command. This one only works for offline migration.